src/crosstargeting_override.props.sample

```diff
@@ -1,37 +1,63 @@
 <Project ToolsVersion="15.0">
+	<!--
+		This file is used to control the platforms compiled by visual studio, and
+			allow for a faster build when testing for a single platform.
+
+			Instructions:
+			1) Copy this file and remove the ".sample" name
+			2) Uncomment and adjust the NugetOverrideVersion property below
+			3) Make sure to do a Rebuild, so that nuget restores the proper packages for the new target
+	-->
+
 	<PropertyGroup>
-		<!--
-			This property allows the override of the nuget local cache.
-			Set it to the version you want to override, used in another app.
-			You will see the override path in the build output.
-			The packages are located under this directory: "%USERPROFILE%\.nuget\packages".
-		-->
-		<!-- <NugetOverrideVersion>1.0.0-dev.1</NugetOverrideVersion> -->
-
-		<!--
-			This property is used to control the platforms compiled by Visual Studio, and
-			allows for a faster build when testing for a single platform.
-
-				Instructions:
-				1) Copy this file and remove the ".sample" name
-				2) Adjust the NugetOverrideVersion property below
-				3) Make sure to do a Rebuild, so that nuget restores the proper packages for the new target
-
-			Available build targets
-
-				uap10.0.18362 (Windows)
-				net6.0-windows10.0.18362 (WinAppSDK Windows)
-				xamarinios10 (iOS)
-				MonoAndroid12.0 (Android 12.0)
-				netstandard2.0 (WebAssembly, Skia)
-				net6.0-ios (.NET 6 iOS)
-				net6.0-android (.NET 6 Android)
-				net6.0-maccatalyst (.NET 6 macOS Catalyst)
-				net6.0-macos (.NET 6 macOS AppKit)
-				xamarinmac20 (macOS)
-
-			Include netstandard2.0 to make sure all projects build
-		-->
-		<!-- <UnoTargetFrameworkOverride>netstandard2.0;uap10.0.18362</UnoTargetFrameworkOverride> -->
+		<!-- See documentation for each property below -->
+
+		<!-- Include netstandard2.0 to make sure all projects build -->
+		<!--<TargetFrameworkOverride>netstandard2.0;net6.0-android</TargetFrameworkOverride>-->
+		<!--<NugetOverrideVersion>2.6.0-dev.34</NugetOverrideVersion>-->
+		<!--<DisableNet6MobileTargets>true</DisableNet6MobileTargets>-->
+
+		<!-- 
+		
+	  ### TargetFrameworkOverride ###
```

Comment:
specially for this one list the full set of examples (and one multi-cases to illustrate the idea) ready to uncomment:
```xml
<!--
	TargetFrameworkOverride lalala~
-->
<!--<TargetFrameworkOverride Condition="''!='hint: uwp'">uap10.0.18362</TargetFrameworkOverride>-->
<!--<TargetFrameworkOverride Condition="''!='hint: wasm,skia'">netstandard2.0</TargetFrameworkOverride>-->
...
<!--<TargetFrameworkOverride>uap10.0.18362;netstandard2.0</TargetFrameworkOverride>-->
```
much better than copy pasting
it is particular annoying to select a chunk of text to copy when it contains `.` or `-` as you just cant double-click to select